uqcsbot/dominos_coupons.py

```diff
@@ -44,50 +48,105 @@ async def dominoscoupons(
         number_of_coupons: int = 5,
         ignore_expiry: bool = True,
         keywords: str = "",
+        source: Literal["both", "couponese", "frugalfeeds"] = "both",
     ):
         """
         Returns a list of dominos coupons
         """
+
         if number_of_coupons < 1 or number_of_coupons > MAX_COUPONS:
             await interaction.response.send_message(
                 content=f"You can't have that many coupons. Try a number between 1 and {MAX_COUPONS}.",
                 ephemeral=True,
             )
             return
+
+        if source != "both":
```

Comment:
This shouldn't be needed any more, given that the source can only be "both", "couponese" or "frugalfeeds" (and should pass type checking).